pywbem_mock/_cimrepository.py

```diff
@@ -1599,39 +1636,39 @@ def GetInstance(self, **params):
                         "does not exist.", iname.classname, iname))
 
         inst = self._get_instance(iname, namespace, instance_store,
-                                  params.get('PropertyList', None),
-                                  params.get('LocalOnly', None),
-                                  params.get('IncludeClassOrigin', None),
-                                  params.get('IncludeQualifiers', None))
+                                  PropertyList,
+                                  LocalOnly,
+                                  IncludeClassOrigin,
+                                  IncludeQualifiers)
 
-        return self._make_tuple([inst])
+        return inst
 
     ####################################################################
     #
     #   DeleteInstance server request
     #
     ####################################################################
 
-    def DeleteInstance(self, **params):
+    def DeleteInstance(self, InstanceName):
         """
-        Implements a mock server responder for
+        Implements a CIM repository responder for
         :meth:`~pywbem.WBEMConnection.DeleteInstance`.
 
-        This deletes a single instance from the mock repository based on the
+        This deletes a single instance from the CIM repository based on the
         iname and namespace parameters.
 
         It does not attempt to delete referenceing instances (associations,
         etc. that reference this instance.)
 
         If the creation class of the instance to be deleted is PG_Namespace or
         CIM_Namespace, then the namespace identified by the 'Name' property
-        of the instance is being deleted in the mock repository in addition to
+        of the instance is being deleted in the CIM repository in addition to
         the CIM instance. The default connection namespace of this faked
         commection cannot be deleted.
         """
 
         namespace = self.namespace
```

Comment:
See previous comment about `namespace = self.namespace`. Only this time, the namespace is (optionally) passed in InstanceName.